test/externalTests/gnosis.sh

```diff
@@ -33,7 +33,7 @@ function gnosis_safe_test
     OPTIMIZER_LEVEL=1
     CONFIG="truffle.js"
 
-    truffle_setup https://github.com/solidity-external-tests/safe-contracts.git development_070
+    truffle_setup https://github.com/cameel/safe-contracts.git external-test-fixes
```

Comment:
Don't you have write access to `solidity-external-tests`?

Reply:
I don't.

Reply:
It seems @ekpyron is the owner on it -- can you add @cameel please?

Reply:
Thanks. Though it's not urgent. This link is just a temporary change that will be removed from the final PR. I was going to submit my fixes to `solidity-external-tests` in PRs like e.g. https://github.com/solidity-external-tests/colonyNetwork/pull/8.

Reply:
Actually these permissions would be handy now because I wanted to pull latest changes from colony into the test repo.

Maybe @chriseth is an owner there too?

Reply:
I just added you there @cameel - I also changed @axic and @chriseth to owners, not sure why that wasn't the case so far. Arguably, we could also just make everyone in the team owner there, not sure...

Reply:
Thanks!

Not sure about owners - limiting permissions is often a good idea and for example I currently don't need that much - but giving everyone on the team write access sounds good.